doc/implicit_conversion.rdoc

```diff
@@ -115,10 +115,19 @@ This class is not Hash-convertible (method +to_hash+ returns non-Hash):
 === Integer-Convertible Objects
 
 An <i>Integer-convertible object</i> is an object that:
+
 * Has instance method +to_int+.
 * The method accepts no arguments.
 * The method returns an object +obj+ for which <tt>obj.kind_of?(Integer)</tt> returns +true+.
 
+The Ruby core classes that satisfy these requirements are:
+
+* Numeric
```

Comment:
I'm not sure if we should put Numeric here since Numeric can't be used by itself, only subclasses of Numeric (which are Integer, Float, Complex, Rational).

Reply:
Removed.